src/backends/tensorflow/base_backend.cc

```diff
@@ -459,7 +459,7 @@ SetStringInputTensor(
   // copy it there.
   auto buffer_memory_type = TRITONSERVER_MEMORY_CPU_PINNED;
   int64_t buffer_memory_type_id = 0;
-  const char* content;
+  const char* content = nullptr;
```

Comment:
Did you find a case where this was necessary?

Reply:
The compiler throws a warning regarding it remaining initialized